habito/habito.py

```diff
@@ -103,9 +115,18 @@ def list():
 
             if quanta >= habit.quantum:
                 column_text = u'\u2713'
-
-            habit_row.append("{0} ({1})".format(column_text, quanta))
-
+                if in_streak:
```

Comment:
I'd suggest we separate the responsibilities between view and model to begin with. Just minimal to get us going with other features. May be it will evolve to complex cli pattern in near future.

Given non continuous nature, it is okay to keep achievements separately. However `list` will likely be the most used command, since `id` is required for other operations. Will computation of `streak` slow it? At check-in, streak is additive if user is on a streak already, if not, streak starts afresh. We've to come up with the right model to store this though.

Are there other *continuous* values we'd want to compute on quantas? E.g. an over all goal (say 42 days from day habit is started, 42 is random), total progress so far, a daily quanta to reach the goal etc.. Just brainstorming to see if we can bucket some of this stats together with streak? Thoughts?